tests/test_order_SolidLiquid.py

```diff
@@ -51,6 +52,36 @@ def test_identical_environments(self):
                 assert comp.cluster_sizes[0] == len(positions)
                 npt.assert_array_equal(comp.num_connections, 12)
 
+    def test_l_zero(self):
```

Comment:
Why is this here rather than the `Steinhardt` class test of spherical harmonic accuracy? I know they use the same underlying code, but I think having them in the same file would make sense from a like tests go together perspective.

Reply:
@b-butler I originally wrote the test in SolidLiquid and only tested the `l = 0` case. When Bradley requested a regression test for more realistic `l`, I copied the `l_zero` test you asked about into the Steinhardt tests and started writing the regression tests. However, I ended up instead just writing a more rigorous test actually recalculating the values using `scipy` for arbitrary `l`, which then superseded the `l = 0` test, so I removed that from Steinhardt and simply forgot to remove it from SolidLiquid. That's now removed, thanks.